console/console.c

```diff
@@ -233,6 +238,9 @@ AttemptGSSAPI(CONSFILE *pcf)
 
     ret = (stmaj == GSS_S_COMPLETE);
     gss_release_name(&stmin, &gss_server_name);
+    if (buf != NULL) {
+	free(buf);
+    }
```

Comment:
There is no need to check if `buf` is `NULL`.  Aside from the fact that, at this point in the code, `buf` "can't" be `NULL`, calling `free()` with `NULL` is a no-op, so the extra cost and complication here (and at `conserver/group.c` line 2018) generates no benefit.

Reply:
fixed, thanks!